### PR TITLE
refactor: remove command-line installer

### PR DESCRIPTION
**Pull request by francesco-racciatti**
**What this PR does / why we need it**:

This PR removes the old command-line installer.

Since we no longer need to release the command-line installer, the only artifact will be the cloudformation patching handle.

So, this PR updates `goreleaser.yml` too, to release the cloudformation patching handle as a zip archive (named `agent-kilt.zip`)

The new golereaser was validated through `goreleaser check`:
```
vagrant@ubuntu22:agent-kilt$ goreleaser check .goreleaser.yml
  • checking                                 path=.goreleaser.yml
  • 1 configuration file(s) validated
  • thanks for using goreleaser!
```

### CODE REVIEW

File: .gitignore
Comment (pgcrooks-sysdig) [inline .gitignore:1]: Worth adding these build artifacts?
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	runtimes/cloudformation/agent-kilt.zip
	runtimes/cloudformation/cfnpatcher/fixtures/respect_ignores/panic_opt_exclude.patched.json
```

Comment (francesco-racciatti) [reply .gitignore:1]: Good catch!

yes for `runtimes/cloudformation/agent-kilt.zip` (and the handler too)

I'm not sure about the other guy. It should be generated by the tests, and it probably should have been included under `patching`. Anyway, we are going to refactor the codebase soon, so I would leave it out for now.

Comment (francesco-racciatti) [reply .gitignore:1]: fixed eb7164ed6a040c6cf2244f5db5da9f11d00d551f

**Review by pgcrooks-sysdig — APPROVED** [review]
LGTM

**Pull request merged**